cloud/linode/fake_linode_test.go

```diff
@@ -186,6 +190,40 @@ func (f *fakeAPI) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 				_, _ = w.Write(rr)
 				return
 			}
+
+			rx, _ = regexp.Compile("/nodebalancers/[0-9]+/firewalls")
```

Comment:
It's weird to ignore the error here - maybe just run `MustCompile` to ensure it passes.